### ISSUE TITLE
Failed to spawn jailbreakd. Every Version, iPhone X 11.2.6

### ISSUE THREAD

**Issue by DarkRagnarok** [pwn20wndstuff/Undecimus#99]
Every version has failed to spawn jailbreakd, have tried all tips but still no success. Tried up to Version 1.1.1 still have no success 1.1.4 same issue

**Comment by slathar** [comment]
same

**Comment by GeoSn0w** [comment]
Using v1.1.0 seems to work.

**Comment by yungmichael** [comment]
My issue fixed y’all may issue using jb remover then fresh Cydia install 

https://twitter.com/pwned4ever/status/1058831893196562432?s=12

**Comment by devcline** [comment]
error with unc0ver 1.1.4 from RC7

```
step 37/56 errno: 2
line 2181, File: ViewController.m,
Function : exploit,
(private/var/temp) or (private/var/temp/jailbreak.pid) == 0
failed to spawn jailbreakd
```

ps: with restore ROOT FS gives me sometimes ( with EL also ) error

```
step 36/56 errno 3
Test : mach_port_valid(task_port)
line 320, File: ViewController.m
function: inject_libiary
failed to inject library
```


ps: home button trick ,respring, reboot ,EL , wait 10 secs,, and clean ram doesn't help

TEMP SOLUTION: back to RC7 tha worked perfects (except failed to purchase free apps while FACE ID is on) 

(i remarked that app revokes auto updates and overwrite boot nonce are ON in RC7 and not in 1.1.4)

ps: i will try this tomorrow 
https://twitter.com/pwned4ever/status/1058831893196562432?s=12

**Comment by Cryptiiiic** [comment]
@DarkRagnarok closing this in around 24 hours if there are no further replies.

**Comment by Cryptiiiic** [comment]
@devcline can you not be annoying please. Don't thumbs down my comment. Also closing this because no replies.

**Comment by Samgisaninja** [comment]
Re-opened because this is still an issue

**Comment by devcline** [comment]
@Cryptiiiic please talk about the dev/bug issues , not about your ego. Thanks

**Comment by findus500** [comment]
Some ideas?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by archange84** [comment]
I have exactly the same problem with my iPhone 8 Plus iOS 11.2.5.
if I can help by doing manipulations.

**Comment by Kudg3** [comment]
Same issue with my iPhone 8, 11.2.1. Haven’t been able to find a fix yet with other versions.

**Comment by pycak1** [comment]
Ditto iPX 11.2

**Comment by pycak1** [comment]
11.2 iPX same issue
Most of time :  line 2182
Sometime :  line 2127
And hardly ever one more, just can’t remember

**Comment by Samgisaninja** [comment]
Fixed in 015cb0a which is bundled in v2.0
